tests/test_api_emmo.py

```diff
@@ -220,15 +221,15 @@ def test_iter(self):
         c.add(n)
         c.add(p, q, rel=mereotopology.hasProperPart)
 
-        elements = set(list(c.iter()))
-        self.assertEqual(elements, {n, p, q})
+        elements = set(c.iter())
+        self.assertSetEqual(elements, {n, p, q})
 
     def test_get_attributes(self):
         """Test getting the attributes."""
         p = math.Real(hasNumericalData=1.2)
         self.assertEqual(
             p.get_attributes(),
-            {math.hasNumericalData: "1.2"}  # TODO type conversion
+            {math.hasNumericalData: {"1.2"}}  # TODO type conversion
```

Comment:
I'd say this would be _"being a reasoner"_, since the dataype of _"hasNumericalData"_ is blank (see attached screenshots). I would rather remove the TODO.

![image](https://user-images.githubusercontent.com/43052541/145594506-c33d79f7-7ee4-4762-ba06-1587a01c02b4.png)
![image](https://user-images.githubusercontent.com/43052541/145594586-5a672d06-3a6f-4b65-8a6e-c1fba408febf.png)

Reply:
I guess I am a reasoner, then. There are worse things to be...
Should we report this to the emmoists?

Reply:
I'd say this is also intentionally done like this on the EMMO. Probably the integer class on EMMO is equivalent to `hasNumericalData some xsd:integer`. I have the feeling that they just want to reuse the relationship for several cases. I also don't think this is an _"illegal"_ thing to do.

Reply:
I think we would like to be a reasoner here but we are not there yet :wink: